docs/integrations.rst

```diff
@@ -213,6 +213,16 @@ PHP Symfony Security Checker
 ----------------------------
 Import results from the PHP Symfony Security Checker.
 
+Probely
+-------
+Synchronize Probely Plus findings with DefectDojo.
+
+To setup this integration set the DefectDojo URL and API key on the Integrations page on Probely. Then, select which Product, Engagement, and, optionally, the Test you want to synchronize to. The API key needs to belong to a staff user.

```

Comment:
I think it could be enough to have a regular user that is authorized for the product and has the "scan_upload" (or create_test) permission in django-admin? Not sure, but could be worthwhile to find a solution that doesn't need staff permissions.

Reply:
We ended up always using the `re_import` endpoint.

Meanwhile we already added the requirement of the staff user, and the indication that we support DD 1.5.X and 1.6.X (while I was writing this 1.7 was launched, and we will test it ASAP and submit a new PR to update the supported versions).

By the way, we tried to implement 2-way sync, but didn't find a way to configure custom webhooks from DefectDojo.

Reply:
We can investigate a way to work with lower privileges, but not sure when. We have plans to add a importer/parser, so we are likely to review this at that moment.

Reply:
No custom webhooks at the time indeed. Contributions are welcome ;-)